wcbench.sh

```diff
@@ -467,7 +532,7 @@ run_cbench()
 }
 
 ###############################################################################
-# Deletes OpenDaylight source (zipped and unzipped)
+# Deletes OpenDaylight source (unzipped so that every time a zipped file isnt downloaded.)
```

Comment:
I don't think these revised docs reflect what the code is actually doing. Could be wrong?